<launch>
    <arg name="rviz" default="true"/>

    <include file="$(find cameras)/launch/d435/d435.launch" >
		<arg name="rviz" value="$(arg rviz)"/>
    </include>  

    <node name="image_saver_node" pkg="image_saver" type="image_saver_node" output="screen">
        <rosparam file="$(find image_saver)/config/d435_image_saver_params.yaml" command="load"/>
    </node>
</launch>
